### ISSUE TITLE
Monorepo nested workspace folders issue

### ISSUE THREAD

**Issue by hassanhe** [tailwindcss/intellisense#127]
Thanks for this great extension ✨
### The Usecase
Working on a monorepo that has multiple packages inside, i usually open the root of the repo alongside some packages inside, as a multi-root workspace.
### The Issue
The extension always picks the outermost workspace folder *(the root)* to run the server on it which results ignoring nested packages configurations although they are **separate apps** with **seperate configs**, right here :
https://github.com/bradlc/vscode-tailwindcss/blob/44131edd936272557548622e6751396b7b59b821/src/extension.ts#L172-L173
### The Solution
- ***Temporarily*** i excluded the root workspace folder and the extension works fine 👍🏼
- I tried on finding the reason for selecting the outermost workspace folder as the convenient server folder, i couldn't *(may you clarify please, thanks)*
- Am wondering if the convention of selecting the server folder would be **The first folder having tailwind config file while searching up for the outer most workspace folder**, i think this solution wouldn't break any current uses for the extention  

Thanks.

**Comment by cesasol** [comment]
I think this might be a related issue but since 3.0 this stopped working on some of my projects. I narrowed down to the fact that I use pnpm for all my projects since when I install  node_modules with yarn or npm it does work as expected.

I already tested with a empty  vscode profile `code --user-data-dir ~/.vscode-test --extensions-dir ~/.vscode-test-extentions` so I was sure that no other plugin was interfering with the test.

For now I'm downgrading to 2.0
